labs/Demos/Cryptol/Caesar.md

```diff
@@ -59,6 +61,7 @@ caesar msg = map rot3 msg
 ```
 
 ```shell
+labs::Demos::Cryptol::Caesar> :s prover=abc
```

Comment:
Let’s stick with z3 everywhere we can. Not everyone can use other provers.

Reply:
Well when I wrote this, Z3's performance was tolerable...but when I ran it last night the `recovery` proofs were especially slow.  Perhaps we could comment this line out and just run it for testing/CI.

Reply:
How about adding a note that `z3` works but is slow, so we're demonstrating with `abc`, and folks can use whichever solver they have installed?